docker/uwsgi.ini

```diff
@@ -0,0 +1,15 @@
+[uwsgi]
+# Python modules integration
+wsgi-file=search/wsgi.py
+callable=APP
+
+# listen for WSGI connections
+socket=0.0.0.0:9000
+# master and workers mode
+master=True
+# process pool size
+processes=2
```

Comment:
Can this be inferred by the number of cores? If so can we do: `processes = number_of_cores - 1`

Reply:
I have no stake in the actual values here as it's mostly for demonstration on how to run the project in a production-like environment. The `sample-configuration` repository will have its copy of this which may use different values.

I'll check what uWSGI supports or uses as a default, but I'd reserve tuning for the actual production configuration: at the moment cores would be shared between different containers anyway as they are all deployed to a single server.

Reply:
(I wanted this to be >1 to be more realistic and discover possible concurrency problems)

Reply:
The default is 1 so not good for the previous comment. Only integers are allowed and `0` actually breaks it, so I'll stick with 2 (or a larger value if we want) for the purpose of Continuous Integration (which doesn't use this yet, but will for smoke tests).